DbAccessor/src/Shared/Utilities.cs

```diff
@@ -0,0 +1,52 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GoRideShare
+{
+    public static class Utilities
+    {
+        // Method that validates headers, outputs the userID and dbToken, returns exception if headers  missing, null if headers are good
+        public static IActionResult ValidateHeaders(IHeaderDictionary headers, out string userId)
+        {
+            userId = string.Empty;
+            
+            // Check for X-User-ID  and X-DbToken headers
```

Comment:
I'm not 100% sure about this one, but I think the headers would have already been consumed by Azure just to hit the endpoint, so I don't know if we need to validate them here?

Can someone confirm one way or another?